packages/std/src/exports.rs

```diff
@@ -1,5 +1,8 @@
 //! exports exposes the public wasm API
-//! allocate and deallocate should be re-exported as is
+//!
+//! cosmwasm_vm_version_1, allocate and deallocate turn into Wasm exports
```

Comment:
Interesting.

I also found the required imports are only those who are called from a valid rust callpath (eg. if the contract never calls `humanize_address` that won't be a required import), which is pretty cool.

The global `cosmwasm_std` exports kind of makes sense, but let me think a bit more on that.